### ISSUE TITLE
Support for Named Entity Recognition Tasks

### ISSUE THREAD

**Issue by anurag899** [langchain-ai/langchain#1006]
@hwchase17 - I have been using langchain for questions answer & summarization tasks and found it quite any helpful. Any plan to add functionality to handle NER tasks. 
If you could suggest if any other resource that can be used for the same. Thanks for reading

**Comment by hwchase17** [comment]
we'll add some examples!

**Comment by anurag899** [comment]
@hwchase17  - Thanks for the update. I have been working on using GPT-3 for named entities task. 
Could you provide me suggestion about how I should deal with map-reduce functionality while parsing large documents?
For example, I have 70-80 pages document and defined an prompt where if model is able to find values from list of given entities then return otherwise leave things as blank.
But in cases where we have large documents we probably get multiple json from each chunks. What could be best way to combine this json and renders at end of function?
@hwchase17  - I could help with the documentation of NER tasks. Let me know how I can be help there.

**Comment by hwchase17** [comment]
not sure i understand the first question

for the documentation, i would add a new section to the "Use Cases" section and add some examples under that

**Comment by anurag899** [comment]
@hwchase17  - Let me rephrase my initial question again -
Suppose we have a large documents with 70-80 pages and are planning to extract the entities.
e.g. - We have a large doc and we divided the texts into chunks 
texts = [["My name is Anurag"],["I work as Software engineer"]]
I have written my prompt like this
prompt = "Use the following pieces of context to extract the values at the end. If you don't find the entity, just return empty value, don't try to return wrong entity.

This should be in the following format:
Context: [Context here]
List of Entities : 
{"Person":[Value],
"Job Title":[Value],
}

Context:
{chunk[idx]}
List of Entities:"

So for the first chunk it would return me - {'Person':"Anurag","Job Title":"___"}
and for the second chunk, it would return me - {"Person":'___","Job Title":"Software engineer"}

So I just want to understand if I am following the right approach. any suggestions to improve the way I am doing.

**Comment by puranjay123** [comment]
@hwchase17 can I work on the NER documentation part in Use case section . I will add some examples for Resume screening as well and add some detailed explanation for the whole example. I will also plan to write a boiler plate code for the NER tasks. 
Assign this issue to me.
